sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1612,7 +1612,9 @@ class Dataset[T] private[sql](
   }
 
   /**
-   * Returns a new Dataset by sampling a fraction of rows.
+   * Returns a new Dataset by sampling a fraction of rows, using a user-supplied seed.
+   * Note: this is NOT guaranteed to provide exactly the fraction specified of the 
```

Comment:
Maybe, we need a newline between description and `Note:` just to be consistent with the others in here and other places such as `functions.scala` if more commits should be pushed.